src/templates/article.jsx

```diff
@@ -1,40 +1,106 @@
-  const { frontmatter, body } = data.mdx;
-  console.log(pageContext);
+  const { frontmatter, body, timeToRead } = data.mdx;
+  const classes = useStyles();
+  const related = [];
+  if(pageContext.prev) related.push(pageContext.prev);
+  if(pageContext.next) related.push(pageContext.next);
   return (
     <Layout>
       <SEO 
         title={frontmatter.title} 
         description={body.substring(0,140).replace(/^#.*?$/gm, "")} 
         article={true} 
       />
-      <Typography variant="h1">{frontmatter.title}</Typography>
-      <Typography variant="subtitle1">{frontmatter.date}</Typography>
+      {/*Title*/}
+      <Typography variant="h3">{frontmatter.title}</Typography>
+      {/*Authors*/}
+      <Grid container alignItems="center">
+        {/*Author Pictures*/}
+        <Grid item>
+          <div className={classes.avatarContainer}>
+          {frontmatter.authors.map(a => 
+            <Avatar alt={a.display_name} src={a.image} title={a.display_name}>{a.username.substring(0,1)}</Avatar>
+          )}
+          </div>
+        </Grid>
+        {/*Author Names/Links*/}
+        <Grid item>
+          <Typography variant="subtitle1">
+            By {frontmatter.authors.map((a, index, array) => 
+              <Fragment>
+                <Link component={GatsbyLink} to={`/author/${a.username}`} className={classes.link}>
+                  {a.display_name}
+                </Link>
+                {index === array.length-2 && " and "}
+                {index < array.length-2 && ", "}
+              </Fragment>
+            )}
+          </Typography>
+        </Grid>
+      </Grid>
+      {/*Date Published*/}
+      <Typography variant="subtitle1" component="time">
+        {" " + frontmatter.date + " "}&middot;{" " + timeToRead + " min read"}
+      </Typography>
       <MDXRenderer>{body}</MDXRenderer>
-      {pageContext.prev && (
-        <Link to={`/articles/${pageContext.prev.node.fields.slug}`}>
-          <Fragment>
-            {"<--"}
-            {pageContext.prev.node.frontmatter.title}
-          </Fragment>
-        </Link>
-      )}
-      {pageContext.next && (
-        <Link to={`/articles/${pageContext.next.node.fields.slug}`}>
-          <Fragment>
-            {"-->"}
-            {pageContext.next.node.frontmatter.title}
-          </Fragment>
-        </Link>
-      )}
+      <p>
+        Tagged: {frontmatter.tags.map(tag => 
+          <TagChip className={classes.chips} name={tag} label={tag} />
+        )}
+      </p>
+      <Grid container justify="space-between">
+        {pageContext.prev && (
+          <Grid item>
+            <Grid container alignItems="center">
+              <ArrowBackIcon />
+              <Link to={`/articles/${pageContext.prev.node.fields.slug}`} component={GatsbyLink} className={classes.link}>
+                {pageContext.prev.node.frontmatter.title}
+              </Link>
+            </Grid>
+          </Grid>
+        )}
+        {pageContext.next && (
```

Comment:
this shows on the left side if there is no previous article. Is that intended?

Reply:
It was intentionally lazy of me but not intended to be a final feature. It's difficult to align things using the grid system when you don't know how many links there will be (prev, next, or prev+next). The pure CSS solution would be to use floats and clearfix, but it doesn't conform to material-ui standards

Reply:
You could use flexbox with space-between like I did in [my attempt to fix this](https://github.com/huntercsblog/huntercsblog.github.io/pull/43/files#diff-8874a6e4b269a1cd7c9ae7090d9d2871R36), but I think an empty grid component would also work and would involve less editing from what you already have.